Sources/PackageDescription/Target.swift

```diff
@@ -275,24 +273,22 @@ public final class Target {
 
     /// Creates a library or executable target.
     ///
-    /// A target can either contain Swift or C-family source files. You can't
-    /// mix Swift and C-family source files within a target. The Swift Package Manager
-    /// considers a target to be an executable target if there's a `main.swift`,
-    /// `main.m`, `main.c`, or `main.cpp` file in the target's directory. It considers
-    /// all other targets to be library targets.
+    /// A target can contain either Swift or C-family source files, but not both. The Swift Package Manager
+    /// considers a target to be an executable target if its directory contains a `main.swift`, `main.m`, `main.c`,
+    /// or `main.cpp` file. The Swift Package Manager considers all other targets to be library targets.
     ///
     /// - Parameters:
     ///   - name: The name of the target.
     ///   - dependencies: The dependencies of the target. A dependency can be another target in the package or a product from a package dependency.
-    ///   - path: The custom path for the target. By default, the Swift Package Manager expects a target's sources to reside at predefined search paths,
+    ///   - path: The custom path for the target. By default, the Swift Package Manager requires a target's sources to reside at predefined search paths;
     ///       for example, `[PackageRoot]/Sources/[TargetName]`.
-    ///       Don't escape the package root; that is, values like `../Foo` or `/Foo` are invalid.
+    ///       Don't escape the package root; for example, values like `../Foo` or `/Foo` are invalid.
     ///   - exclude: A list of paths to files or directories that the Swift Package Manager shouldn't consider to be source or resource files.
-    ///       This path is relative to the target's directory.
```

Comment:
This makes it sound kind of unrelated to the former, doesn't it? How about "Each of these paths" or something like that?

Reply:
How about `Paths are relative to the target's directory.`?

Reply:
That wording sounds good.

Reply:
Done.